### PR TITLE
Fix!: Version fix for main package 0.4.0 -> 0.4.1

### PR DESCRIPTION
**Pull request by schmidmt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [rust-0.4.1] - 2023-10-16
```
Comment (Swandog) [inline CHANGELOG.md:10]: You need to add the entry at the bottom of this file to match the `rust-0.4.1` version

**Review by Swandog — CHANGES_REQUESTED** [review]
One thing needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -141,6 +146,7 @@ Initial release on [PyPi](https://pypi.org/)
 Initial release on [crates.io](https://crates.io/)
 
 [unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.0...HEAD
+[rust-0.4.0]: https://github.com/promised-ai/lace/compare/rust-0.4.0...rust-0.4.1
```
Comment (Swandog) [inline CHANGELOG.md:149]: The line above should change too

Comment (schmidmt) [reply CHANGELOG.md:149]: ✔️

File: CHANGELOG.md [hunk 3]
```diff
@@ -140,7 +145,8 @@ Initial release on [PyPi](https://pypi.org/)
 
 Initial release on [crates.io](https://crates.io/)
 
-[unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.0...HEAD
+[unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.1...HEAD
```
Comment (Swandog) [inline CHANGELOG.md:148]: The latest tag is `rust-0.4.1`, not `python-0.4.1`...the python tag doesn't exist

Comment (schmidmt) [reply CHANGELOG.md:148]: 🤦

**Review by Swandog — APPROVED** [review]
Approved due to work done while pairing

**Pull request merged**